### PR TITLE
Add new 'customer-ssp' folder with new index and add to parent index

### PR DESCRIPTION
**Pull request by wylie-ucb**

### CODE REVIEW

File: docs/cloud/aws/lca1/customer-ssp/index.md
Comment (wylie-ucb) [inline docs/cloud/aws/lca1/customer-ssp/index.md:77]: Be consistent with periods in bullet lists

File: docs/cloud/aws/lca1/index.md
Comment (wylie-ucb) [inline docs/cloud/aws/lca1/index.md:55]: Make this match the title of the actual page (SSP)

File: docs/cloud/aws/lca1/customer-ssp/index.md
Comment (wylie-ucb) [inline docs/cloud/aws/lca1/customer-ssp/index.md:31]: Add section about securing the Root account.

File: docs/cloud/aws/lca1/customer-ssp/index.md
Comment (wylie-ucb) [inline docs/cloud/aws/lca1/customer-ssp/index.md:65]: extra line

File: docs/cloud/aws/lca1/customer-ssp/index.md
Comment (wylie-ucb) [inline docs/cloud/aws/lca1/customer-ssp/index.md:40]: CFS, not CSF

File: docs/cloud/aws/lca1/customer-ssp/index.md
Comment (wylie-ucb) [inline docs/cloud/aws/lca1/customer-ssp/index.md:17]: Do we really want to make the statement about Highly Confidential?  I wasn't sure how we want to state this.

File: docs/cloud/aws/lca1/customer-ssp/index.md
Comment (wylie-ucb) [inline docs/cloud/aws/lca1/customer-ssp/index.md:8]: OIT-Sec guides and evaluates.  The PO is responsible for the approval it sounds like.  This is muddy territory.

File: docs/cloud/aws/lca1/customer-ssp/index.md
Comment (wylie-ucb) [inline docs/cloud/aws/lca1/customer-ssp/index.md:34]: 'designed to enable support' might be better.

File: docs/cloud/aws/lca1/customer-ssp/index.md
Comment (wylie-ucb) [inline docs/cloud/aws/lca1/customer-ssp/index.md:80]: change 'approved' to 'reviewed'

File: docs/cloud/aws/lca1/customer-ssp/index.md
Comment (wylie-ucb) [reply docs/cloud/aws/lca1/customer-ssp/index.md:8]: Removed 'approval'

File: docs/cloud/aws/lca1/customer-ssp/index.md
Comment (wylie-ucb) [reply docs/cloud/aws/lca1/customer-ssp/index.md:34]: fixed

File: docs/cloud/aws/lca1/customer-ssp/index.md
Comment (wylie-ucb) [reply docs/cloud/aws/lca1/customer-ssp/index.md:40]: fixed

File: docs/cloud/aws/lca1/customer-ssp/index.md
Comment (wylie-ucb) [reply docs/cloud/aws/lca1/customer-ssp/index.md:77]: fixed

File: docs/cloud/aws/lca1/customer-ssp/index.md
Comment (wylie-ucb) [reply docs/cloud/aws/lca1/customer-ssp/index.md:65]: removed

File: docs/cloud/aws/lca1/customer-ssp/index.md
Comment (wylie-ucb) [reply docs/cloud/aws/lca1/customer-ssp/index.md:80]: fixed

File: docs/cloud/aws/lca1/index.md
Comment (wylie-ucb) [reply docs/cloud/aws/lca1/index.md:55]: fixed

File: docs/cloud/aws/lca1/customer-ssp/index.md
Comment (wylie-ucb) [reply docs/cloud/aws/lca1/customer-ssp/index.md:31]: done

**Review by jarmbruster74 — APPROVED** [review]
Looks great overall.  Let's capture the idea of adding in specific control references in a later ticket - I think this gets us to a clear starting point for our next set of discussions with customers.

File: docs/cloud/aws/lca1/customer-ssp/index.md
Comment (wylie-ucb) [reply docs/cloud/aws/lca1/customer-ssp/index.md]: I updated the LCA1 support indicating that highly confidential is on the horizon.

File: docs/cloud/aws/lca1/customer-ssp/index.md
Comment (hexmoose) [inline docs/cloud/aws/lca1/customer-ssp/index.md:8]: I think they generally go by "OIT Security" not "OIT IT Security".

File: docs/cloud/aws/lca1/customer-ssp/index.md
Comment (hexmoose) [inline docs/cloud/aws/lca1/customer-ssp/index.md:47]: Should we also mention group audit logging (via Grouper) here?

File: docs/cloud/aws/lca1/customer-ssp/index.md
Comment (hexmoose) [inline docs/cloud/aws/lca1/customer-ssp/index.md:76]: I would change to "you have a private subnet with an encrypted connection to the CU Boulder campus".

File: docs/cloud/aws/lca1/customer-ssp/index.md
Comment (hexmoose) [inline docs/cloud/aws/lca1/customer-ssp/index.md:79]: Should we link to https://curc--382.org.readthedocs.build/en/382/cloud/aws/lca1/networking.html for more information?

File: docs/cloud/aws/lca1/customer-ssp/index.md
Comment (hexmoose) [inline docs/cloud/aws/lca1/customer-ssp/index.md:97]: This seems out of order. Won't they need their account first in order to do the security work as part of their SSP? I'm thinking something like:
1. Determine Data Classification.
2. Understand Shared Responsibility model.
3. Get an account on LCA1 (questionnaire is part of that).
4. Secure your workload and create your SSP. (We should either expand more on this or link to OIT Security documentation.)
5. Get a review from OIT Security of your SSP. (We should either expand more on this or link to OIT Security documentation.)
6. After OIT Security review and Product Owner approval, start operating your secure workload.

File: docs/cloud/aws/lca1/customer-ssp/index.md
Comment (hexmoose) [inline docs/cloud/aws/lca1/customer-ssp/index.md:96]: I'm not sure if this is the time, but the process for doing and reviewing and SSP is still pretty poorly documented. Are we expanding on that here or asking OIT Security to write up a page on the website? If I were a customer, I'd be lost at this point.

File: docs/cloud/aws/lca1/customer-ssp/index.md
Comment (wylie-ucb) [reply docs/cloud/aws/lca1/customer-ssp/index.md:8]: Looking here: https://oit.colorado.edu/services/it-security it looks like they go by "IT Security".  How about `OIT's [IT Security](link)`?

File: docs/cloud/aws/lca1/customer-ssp/index.md
Comment (wylie-ucb) [reply docs/cloud/aws/lca1/customer-ssp/index.md:47]: I will add something like "The role we provide you is set up with temporary short-term credentials to help you meet compliance."

File: docs/cloud/aws/lca1/customer-ssp/index.md
Comment (wylie-ucb) [reply docs/cloud/aws/lca1/customer-ssp/index.md:76]: Please propose better verbiage.  I don't want to go back and forth trying to see what you want here.

File: docs/cloud/aws/lca1/customer-ssp/index.md
Comment (wylie-ucb) [reply docs/cloud/aws/lca1/customer-ssp/index.md:96]: Agree 100%, we have nothing to point to.  This is one of the major gaps identified as part of the story AC.  If you have suggestions to expand, please let me know so we can discuss.  I'm not comfortable speculating on the process workflow that OIT-Sec is going to require from our users.  I'm not sure they even know.

File: docs/cloud/aws/lca1/customer-ssp/index.md
Comment (wylie-ucb) [reply docs/cloud/aws/lca1/customer-ssp/index.md:9]: > Can we point to something on the security site that defines the control set? If not, we need to identify this gap for security - we aren't the authority here and this can change over time.

I looked for a documented statement from IT Security that mentioned this, but did not find a written statement that NIST 800-171 R2 was the official spec.  This has been captured as a gap in the story.  I'll incorporate the link references you added to this section.

File: docs/cloud/aws/lca1/customer-ssp/index.md
Comment (hexmoose) [reply docs/cloud/aws/lca1/customer-ssp/index.md:8]: You pick the one that you think is the most right. Either way is fine.

Comment (wylie-ucb) [reply docs/cloud/aws/lca1/customer-ssp/index.md:8]: updated

File: docs/cloud/aws/lca1/customer-ssp/index.md
Comment (wylie-ucb) [reply docs/cloud/aws/lca1/customer-ssp/index.md:76]: updated

File: docs/cloud/aws/lca1/customer-ssp/index.md
Comment (wylie-ucb) [reply docs/cloud/aws/lca1/customer-ssp/index.md:79]: Good one.  I added a new paragraph to the bottom of the section.

File: docs/cloud/aws/lca1/customer-ssp/index.md
Comment (wylie-ucb) [reply docs/cloud/aws/lca1/customer-ssp/index.md:97]: updated

File: docs/cloud/aws/lca1/customer-ssp/index.md
Comment (wylie-ucb) [reply docs/cloud/aws/lca1/customer-ssp/index.md:96]: No change unless you have something we should add.

File: docs/cloud/aws/lca1/customer-ssp/index.md
Comment (wylie-ucb) [reply docs/cloud/aws/lca1/customer-ssp/index.md]: Updated all references to "IT Security" or OIT's "IT Security" to be consistent.

File: docs/cloud/aws/lca1/customer-ssp/index.md
Comment (wylie-ucb) [reply docs/cloud/aws/lca1/customer-ssp/index.md:74]: I'm not able to confirm this one.  AFAIK CloudWatch logs are stored in the CloudWatch service at the account level.  Users can set retention, but that's it.  Not centralized with our ControlTower logs.

File: docs/cloud/aws/lca1/customer-ssp/index.md
Comment (wylie-ucb) [reply docs/cloud/aws/lca1/customer-ssp/index.md]: updated

File: docs/cloud/aws/lca1/customer-ssp/index.md
Comment (wylie-ucb) [reply docs/cloud/aws/lca1/customer-ssp/index.md:74]: Updated per after-party discussion this morning.

**Pull request merged**
